signing_service/README.md

```diff
@@ -34,10 +34,15 @@ python -m pytest -p no:django .
 
 To run SigningService you need the following:
 
-- Address and port of a Concent cluster to connect to. If port is omitted, default `9055` will be used.
+- Address and port of a MiddleMan service on a Concent cluster to which SigningService wants to connect to.
+  If port is omitted, default `9055` will be used.
```

Comment:
``default `9055` will be used.`` -> ``9055` is used by default.``

Reply:
Done :+1: